.travis.yml

```diff
@@ -18,11 +18,11 @@ addons:
 
 before_install:
   - |
-      git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(.md)|(.txt)|(.png)|(.jpg)|(.gif)|^(LICENSE)|^(docs)'
+      git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(.md)|(.txt)|(.png)|(.jpg)|(.gif)|^(LICENSE)|^(docs)|^(.templates)'
```

Comment:
Not your fault, but I realised all the `.`s need to be escaped (otherwise they match any character). Could you help fix all of them? Add a backslash in front of all of them, i.e. `(\.md)|(\.txt)|...|^(\.templates))`